### ISSUE TITLE
GPU Conv Fusion Code is disabled in grappler remapper.cc

### ISSUE THREAD

**Issue by OuHangKresnik** [tensorflow/tensorflow#32781]
The grappler remapper optimizer disabled the ability to fuse conv related op in GPU.

Since related code is inside 
#ifndef INTEL_MKL
#endif 

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/grappler/optimizers/remapper.cc

**Comment by gadagashwini** [comment]
@OuHangKresnik, Please elaborate the issue with context. Thanks!

**Comment by OuHangKresnik** [comment]
The context is then using GPU, this optimization seems to be not happening

**Comment by gadagashwini** [comment]
@OuHangKresnik, Will it be possible to provide the sample standalone code reproduce the issue reported here. Thanks!

**Comment by OuHangKresnik** [comment]
With any network that contains Convolution and BatchNorm stuff.

config = tf.ConfigProto(device_count = {'GPU': 0})

With above configuration, the _FusedConv2D operator will show up (in CPU mode).
Without it, then _FusedConv2D won't show up (in GPU mode)

**Comment by jvishnuvardhan** [comment]
@OuHangKresnik Sorry for the late response. Is this still an issue? Thanks!

**Comment by jvishnuvardhan** [comment]
This is a stale issue. Please check the issue with latest TensorFlow. If the issue still persists in the newer version of TF, please feel free to reopen it by providing details about the issue and a standalone code to reproduce the issue. Thanks!

**Comment by CaptainDuke** [comment]
I got same problem. 
I'm testing conv+bias_add+relu case, which should be fused as shown in
https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/core/grappler/optimizers/remapper_test.cc#L294

However, fused ops only exists when using CPU, but disappear when using GPU. Seems like it doesn't work when using GPU ?
[test_conv.zip](https://github.com/tensorflow/tensorflow/files/4034675/test_conv.zip)
